express-delete/index.js

```diff
@@ -0,0 +1,41 @@
+const express = require('express');
+const app = express();
+const grades = {
+  12: {
+    id: 12,
+    name: 'Tim Berners-Lee',
+    course: 'HTML',
+    score: 95
+  },
+  47: {
+    id: 47,
+    name: 'Brendan Eich',
+    course: 'JavaScript',
+    score: 100
+  },
+  273: {
+    id: 273,
+    name: 'Forbes Lindsay',
+    course: 'JavaScript',
+    score: 92
+  }
+};
+
+app.get('/api/grades', (req, res) => {
+  const gradesArray = [];
+  for (const items in grades) {
+    gradesArray.push(grades[items]);
+  }
+  res.jsonp((gradesArray));
```

Comment:
`jsonp()` isn't something that's really used anymore. instead we just stick with `.json()`